src/plugins/workspace/public/components/workspace_updater/workspace_updater.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useCallback, useEffect, useState } from 'react';
+import { EuiPage, EuiPageBody, EuiPageHeader, EuiPageContent } from '@elastic/eui';
+import { useObservable } from 'react-use';
+import { i18n } from '@osd/i18n';
+import { of } from 'rxjs';
+
+import { WorkspaceAttribute } from 'opensearch-dashboards/public';
+import { useOpenSearchDashboards } from '../../../../../../src/plugins/opensearch_dashboards_react/public';
```

Comment:
This import path can be shortened to:
`import { useOpenSearchDashboards } from '../../../opensearch_dashboards_react/public';`